tf/gcp/main.tf

```diff
@@ -123,12 +123,23 @@ resource "google_compute_instance" "fn" {
       agent = true
     }
   }
+  provisioner "file" {
+    source      = "files/reboot.sh"
+    destination = "/home/gce-user/reboot.sh"
+    connection {
+      host  = "${google_compute_instance.fn.network_interface.0.access_config.0.nat_ip}"
+      type  = "ssh"
+      user  = "gce-user"
+      agent = true
+    }
+  }
 
   provisioner "remote-exec" {
     inline = [
+      "sudo sed -i /etc/selinux/config -r -e 's/^SELINUX=.*/SELINUX=disabled/g'",
```

Comment:
1. why do you need to disable selinux?  

2. no need to do reboot, disable it in current session using `sudo setenforce 0` and do the sed, so the next time after reboot, it should be disabled automatically.

Reply:
1. selinux prevent node_exporter to start properly, this is the issue we discovered on DO project. 
2. Sure